.github/workflows/tests.yml

```diff
@@ -82,6 +82,9 @@ jobs:
           echo "::group::Pnpm"
           pnpm install
           echo "::endgroup::"
+          echo "::group::Composer"
+          composer install --working-dir=tools/php-test-env
+          echo "::endgroup::"
```

Comment:
This probably needs to go after the bit just below that's clearing the Composer cache if we want WorDBless nightlies.

You'll probably also need to put code like this before this bit too for that case.
https://github.com/Automattic/jetpack/blob/d1a3fa1a35394e2a2816859939282ea500f04c82/.github/workflows/tests.yml#L199-L209
(and could probably get rid of it there then)